src/CSSList/CSSList.php

```diff
@@ -4,8 +4,7 @@
 
 namespace Sabberworm\CSS\CSSList;
 
-use Sabberworm\CSS\Comment\Comment;
-use Sabberworm\CSS\Comment\Commentable;
```

Comment:
I don't know why the fixer didn't complain about this unsed `use` (`CSSListItem` extends `Commentable`, as of #1212, but when that change was made, `Commentable` was no longer directly referenced in this file).